ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/provision/cucushift-installer-rehearse-gcp-ipi-ovn-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-ovn-provision
```

Comment:
this is not `vsphere`, should be `cucushift-installer-rehearse-gcp-ipi-ovn-provision`

Reply:
Thanks! my bad